moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -113,6 +116,12 @@ export const ProjectsListViewQueryConf = {
       searchable: true,
       sortable: false,
       label: "eCapris Subp.",
+      filter: value =>
```

Comment:
cool to see how this plugs into the filter component!

Reply:
thanks, John!